hikari/users.py

```diff
@@ -512,6 +512,11 @@ def default_avatar_url(self) -> files.URL:
             file_format="png",
         )
 
+    @property
+    def display_avatar_url(self) -> files.URL:
+        """Display avatar URL for this user."""
+        return self.avatar_url or self.default_avatar_url
```

Comment:
Deleted the wrong comment while doing my review by mistake (*Original comment by snab*):

"Missing test coverage"